src/core/load_balancing/rls/rls.cc

```diff
@@ -1194,18 +1218,20 @@ RlsLb::Cache::Entry::Entry(RefCountedPtr<RlsLb> lb_policy,
           lb_policy_->cache_.lru_list_.end(), key)) {}
 
 void RlsLb::Cache::Entry::Orphan() {
+  // We should be holding RlsLB::mu_.
   GRPC_TRACE_LOG(rls_lb, INFO)
       << "[rlslb " << lb_policy_.get() << "] cache entry=" << this << " "
       << lru_iterator_->ToString() << ": cache entry evicted";
   is_shutdown_ = true;
   lb_policy_->cache_.lru_list_.erase(lru_iterator_);
-  lru_iterator_ = lb_policy_->cache_.lru_list_.end();  // Just in case.
+  lru_iterator_ = lb_policy_->cache_.lru_list_.end();
+  CHECK(child_policy_wrappers_.empty());
+  // Just in case.
```

Comment:
What does this comment mean?  We do need to cancel the backoff timer here if it's running.

Reply:
ah, it got moved